### ISSUE TITLE
bug with las conversion

### ISSUE THREAD

**Issue by MapsterMapperDev** [jawline/LAS-3d-printer#1]
I have las files created from ouster lidar sensors using a mobile mapping vehicle. When I attempt any method of converting my first test.las file of a smaller road which is about 500MB in size the exported png/bin/stl file ends up with a size of 1KB and is just a long narrow rectangle. I downloaded the sample data linked in the readme.md and that data converts really well. I don't know if there is a way we can get this road data converted correctly as this data was recorded from the ground and not the air. If there is any methods I can try for converting this data or if you would like a copy of my test.las file it would be greatly appriciated.

**Comment by jawline** [comment]
Hey, this sounds pretty interesting. I wonder if it's a fundamental failure
or just something to do with scaling.

I don't think ground capture Las will render well out of the box because we
are filling in any overhangs as we generate the terrain map. I wouldn't
expect a rect unless the Las was fully enclosed though. This also isn't
intractable, it would be pretty straightforward to use a voxel grid rather
than a 2d one.

Could you send the las? I'm away right now and not sure I have enough local
compute but if my machine can process it I'm happy to take a look.

**Comment by MapsterMapperDev** [comment]
Yes of course. That is what we figured was most likely the issue since it is taken from the ground. We would very much like to work with you on this if we can come up with a method of getting this data into a format that we can use in blender. We are a 3D mobile mapping company, we have an online viewer for our point clouds but the software is proprietary from Bentley. If we can get this data into some kind of model that we can then manipulate like an STL file this would be superb. My boss knows more about this than I do as he's the LiDar specialist. So if there is any other information you need such as GPS metadata, or even the raw pcap and json data from the ouster sensor we'd be happy to oblige. You can find the test las file I was attempting [here.](https://cms.mapsteronline.com/test/test-las.zip) Any and all feedback is appreciated.
